backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/CAdapterGenerator.kt

```diff
@@ -848,7 +848,17 @@ internal class CAdapterGenerator(val context: Context) : DeclarationDescriptorVi
         output("typedef unsigned long long ${prefix}_KULong;")
         output("typedef float              ${prefix}_KFloat;")
         output("typedef double             ${prefix}_KDouble;")
-        output("typedef float __attribute__ ((__vector_size__ (16))) ${prefix}_KVector128;")
+
+        if (context.config.target.family == Family.MINGW) {
+            output("#ifndef _MSC_VER")
+            output("typedef float __attribute__ ((__vector_size__ (16))) ${prefix}_KVector128;")
+            output("#else")
+            output("typedef __m128 ${prefix}_KVector128;")
```

Comment:
Does using `__m128` type require a header to be included?
According to the documentation, it should be `xmmintrin.h`.